src/System.Globalization/tests/DateTimeFormatInfo/DateTimeFormatInfoCalendarWeekRule.cs

```diff
@@ -45,33 +45,18 @@ public void TestInvalidValue()
             });
         }
 
-        // TestLocaleFirstDay: Verify value of property CalendarWeekRule for specific locale
-        [Fact]
-        public void TestLocaleFirstDay()
-        {
-            CultureInfo myTestCulture = new CultureInfo("en-US");
-            DateTimeFormatInfo dti = myTestCulture.DateTimeFormat;
-            CalendarWeekRule actual = dti.CalendarWeekRule;
-            Assert.Equal(CalendarWeekRule.FirstDay, actual);
-        }
-
-        // TestLocaleFirstFourDayWeek: Verify value of property CalendarWeekRule for specific locale
-        [Fact]
-        public void TestLocaleFirstFourDayWeek()
+        // Verify value of property CalendarWeekRule for specific locales
+        [Theory]
+        [InlineData("en-US")]
+        [InlineData("br-FR")]
+        public void TestLocale(string localeName)
```

Comment:
@steveharter added this test, so he can comment as he wants.

IMO, adding more cultures won't really help increase the coverage since we are getting this data from ICU. We just need to test that we are getting the values correctly from ICU. Also, adding more cultures increases the chance that we will get conflicting values between all the differing systems we are testing on.

I think the two cultures are sufficient for testing coverage, so the question becomes, should I split the test back out into 2 separate Facts instead of using a Theory.